lib/optimizely/config_manager/http_project_config_manager.rb

```diff
@@ -30,7 +30,7 @@ module Optimizely
   class HTTPProjectConfigManager < ProjectConfigManager
     # Config manager that polls for the datafile and updated ProjectConfig based on an update interval.
 
-    attr_reader :stopped
+    attr_reader :blocking_timeout, :error_handler, :logger, :notification_center, :polling_interval, :stopped
```

Comment:
I see no issue to make reader attribute, but plz specify reason if those are helping in unit testing.

Reply:
Yes, helps with unit testing.